ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -1061,6 +1068,14 @@ namespace EditMode {
          * @internal
          */
         tools?: Tools;
+         /**
+         * Events attached to the component : `activated`, `deactivated`.
+         *
+         * Try it:
+         *
+         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/edit-mode/events/ | Activated / Deactivated event }
+         */
+         events?: Record<string, Function>;
```

Comment:
+ Too many spaces here. :)
```suggestion
        /**
        * Events attached to the component : `activated`, `deactivated`.
        *
        * Try it:
        *
        * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/edit-mode/events/ | Activated / Deactivated event }
        */
        events?: Record<string, Function>;
```